ios/brave-ios/Sources/BrowserMenu/BrowserMenuStrings.swift

```diff
@@ -0,0 +1,124 @@
+// Copyright (c) 2024 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+import Foundation
+@_exported import Strings
+
+extension Strings {
+  enum BrowserMenu {
+    public static let allSettingsButtonTitle = NSLocalizedString(
+      "BrowserMenu.allSettingsButtonTitle",
+      bundle: .module,
+      value: "All Settings",
+      comment: "The title of a button that presents the settings screen"
+    )
+    public static let myActions = NSLocalizedString(
+      "BrowserMenu.myActions",
+      bundle: .module,
+      value: "My Actions",
+      comment:
+        "The title above a list of the users selected browser actions such as bookmarks, history, etc."
+    )
+    public static let editButtonTitle = NSLocalizedString(
+      "BrowserMenu.editButtonTitle",
+      bundle: .module,
+      value: "Edit",
+      comment:
+        "The title of a button that shows a screen that lets the user customize which actions are visible on the menu"
+    )
+    public static let vpnButtonTitle = NSLocalizedString(
+      "BrowserMenu.vpnButtonTitle",
+      bundle: .module,
+      value: "VPN",
+      comment: "A title on a button that lets the user switch the active VPN's selected region."
+    )
+    public static let customizeTitle = NSLocalizedString(
+      "BrowserMenu.customizeTitle",
+      bundle: .module,
+      value: "Customize",
+      comment: "A title shown at the top of the edit/cutsomize actions screen."
+    )
+    public static let doneButtonTitle = NSLocalizedString(
+      "BrowserMenu.doneButtonTitle",
+      bundle: .module,
+      value: "Done",
+      comment: "A button title that when tapped dismisses the customize screen."
+    )
+    public static let visibleActionsTitle = NSLocalizedString(
+      "BrowserMenu.visibleActionsTitle",
+      bundle: .module,
+      value: "All Actions",
+      comment: "A title shown above the list of actions that are visible on the menu."
+    )
+    public static let hiddenActionsTitle = NSLocalizedString(
+      "BrowserMenu.hiddenActionsTitle",
+      bundle: .module,
+      value: "Available Actions",
+      comment:
+        "A title shown above the list of actions that are hidden on the menu but available for the user to make visible."
+    )
+    public static let showAllButtonTitle = NSLocalizedString(
+      "BrowserMenu.showAllButtonTitle",
+      bundle: .module,
+      value: "Show All…",
```

Comment:
nit: The `…` is a special Unicode character. It should be three separate dots like: `...`?

Reply:
Yep thats on purpose, specifically want to use the ellipsis character in this scenario (`…` vs `...`)